pkg/addons/assets/neuron-device-plugin.yaml

```diff
@@ -113,7 +113,7 @@ spec:
                       - trn1.2xlarge
                       - trn1.32xlarge
       containers:
-        - image: "public.ecr.aws/neuron/neuron-device-plugin:1.5.3.0"
+        - image: "public.ecr.aws/neuron/neuron-device-plugin:1.9.3.0"
```

Comment:
Please change this to the latest 2.1.2.0

Reply:
updated to 2.1.2.0